src/Employer/UnitTests/Employer.Web/Orchestrators/VacancyPreviewOrchestratorTests.cs

```diff
@@ -91,5 +91,58 @@ public VacancyPreviewOrchestratorTests()
             response.Data.IsSubmitted.Should().Be(shouldBeSubmitted);
             response.Data.HasLegalEntityAgreement.Should().Be(hasLegalEntityAgreement);
         }
+
+        [Fact]
+        public async Task RejectJobAdvertAsync_ShouldSendCommand()
+        {
+            //Arrange            
+            var user = new VacancyUser { Email = "advert@advert.com", Name = "advertname", UserId = "advertId" };
+
+            var vacancy = new Vacancy
+            {
+                EmployerAccountId = "ABCDEF",
+                AccountLegalEntityPublicHashedId = "XVYABD",
+                Id = Guid.Parse("84af954e-5baf-4942-897d-d00180a0839e"),
+                Title = "has a value",
+                NumberOfPositions = 1,
+                EmployerLocation = new Address { Postcode = "has a value" },
+                ShortDescription = "has a value",
+                ProgrammeId = "has a value",
+                Wage = new Wage { Duration = 1, WageType = WageType.FixedWage },
+                LegalEntityName = "legal name",
+                EmployerNameOption = EmployerNameOption.RegisteredName,
+                StartDate = DateTime.Now,
+                Status = VacancyStatus.Review,
+                VacancyReference = 123456
+            };
+
+            var rejectJobAdvertViewModel = new RejectJobAdvertViewModel { RejectJobAdvert = true, VacancyId = vacancy.Id, EmployerAccountId = "ABCDEF" };
+
+            _mockRecruitVacancyClient.Setup(x => x.GetVacancyAsync(vacancy.Id)).ReturnsAsync(vacancy);
+            _mockRecruitVacancyClient.Setup(x => x.GetEmployerDescriptionAsync(vacancy)).ReturnsAsync("employer description");
+            _mockRecruitVacancyClient.Setup(x => x.GetEmployerNameAsync(vacancy)).ReturnsAsync("employer name");
+            _mockRecruitVacancyClient.Setup(c => c.Validate(vacancy, It.IsAny<VacancyRuleSet>())).Returns(new EntityValidationResult());
+
+            var geocodeImageService = new Mock<IGeocodeImageService>();
+            var externalLinks = new Mock<IOptions<ExternalLinksConfiguration>>();
+            var mapper = new DisplayVacancyViewModelMapper(geocodeImageService.Object, externalLinks.Object, _mockRecruitVacancyClient.Object);
+            var shouldBeRejected = true;
```

Comment:
no need for this local variable, this pattern used in the above test for multiple test cases

Reply:
true, removed 👍